main/resource-access.bs

```diff
@@ -75,6 +75,291 @@ Issue: Write Linked Data Platform section.
+`Link: <https://server.example/acls/resource.acl>; rel="acl"`
+</p>
+
+The following table lists [[#rm-reserved]] and the associated link relation URIs
+that are used for discovery. Other metadata types and relations may also be
+used, and may be added to the reserved set in the future.
+
+<table class="data" align="left">
+  <colgroup class="header"></colgroup>
+  <colgroup span="2"></colgroup>
+  <thead>
+    <tr>
+      <th>Metadata Type</th>
+      <th>Link Relation</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>[[#rm-wac]]</td>
+      <td>```"acl"``` or ```http://www.w3.org/ns/solid/terms#acl```</td>
+    </tr>
+    <tr>
+      <td>[[#rm-description]]</td>
+      <td>```"describedby"``` or
+```https://www.w3.org/ns/iana/link-relations/relation#describedby```</td>
+    </tr>
+    <tr>
+      <td>[[#rm-shape]]</td>
+      <td>```http://www.w3.org/ns/solid/terms#shape```</td>
+    </tr>
+    <tr>
+      <td>[[#rm-managed]]</td>
+      <td>```http://www.w3.org/ns/solid/terms#servermanaged```</td>
+    </tr>
+    <tr>
+      <td>[[#rm-configuration]]</td>
+      <td>```http://www.w3.org/ns/solid/terms#configuration```</td>
+    </tr>
+  </tbody>
+</table>
+
+#### Discovery of Annotated Resource #### {#rm-annotated}
+
+Certain metadata resource types require the Solid server to link back to the
+annotated resource that the metadata is directly associated with, via Link
+headers. In these instances, the link relation ```rel=describes``` MUST be used.
+
+<p class="example">
+A metadata resource `<https://server.example/metadata/resource.meta>` is
+directly associated with `<https://server.example/resource.ttl>`. The metadata
+type for `resource.meta` requires linking back to the annotated resource, which
+is `resource.ttl`. `resource.meta` must have the following included in its
+`Link` headers:<br/>
+`Link: <https://server.example/resource.ttl>; rel="describes"`
+</p>
+
+### Metadata Characteristics ### {#rm-characteristics}
+
+A given resource MAY Link to metadata on a different server under a different
+authority, per the configuration of the Solid server on which that resource
+resides.
+
+Metadata resources that reside on a Solid server MUST adhere to the same
+interaction model used by other standard Solid resources, except where specified
+in the definition of the associated metadata type in [[#rm-reserved]].
+
+A metadata resource MUST be deleted by the Solid server when the resource it is
+directly associated with is also deleted and the Solid server is authoritative
+for both resources.
+
+### Reserved Metadata Types ### {#rm-reserved}
+
+#### Web Access Control #### {#rm-wac}
+
+ACL resources as defined by [[!WAC]] MUST be supported as a resource metadata
+type by Solid servers.
+
+The ACL metadata resource directly associated with a given resource is
+discovered by the client via the `rel="acl"` `Link` relation in a `Link` header.
+
+A given Solid resource MUST NOT be directly associated with more than one ACL
```

Comment:
Remove "directly" (inheritance)?

Reply:
My mental model is that there is exactly one ACL resource (or handle, if you will) that is allocated to a give resource. If it doesn't exist, the inheritance kicks in. So "associated with" means there is a dedicated URL to fulfill the role of ACL.

Reply:
@ericprud has it right - i also explained in [this comment](https://github.com/solid/specification/pull/156/files/4c4dde9fe6352dd5802fd4b4b4d0fc98c7de7657#r397562513). Note that we do explicitly say in the individual metadata type sections that the relationships are 1:1. We could consider moving that up to the general section so its all stated togther.